appcelerator/titanium_mobile
# feat(android): add missing options to Intl.DateTimeFormat.resolvedOptions()

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28251

**Summary:**
- Adds missing properties returned by `Intl.DateTimeFormat.resolvedOptions()`.
- Note that this returns "timeZone" which is the only API providing this info on both Android and iOS.

**Test:**
1. Build and run the below on Android.
2. Verify an alert dialog appears displaying the device's assigned time zone.
  2.1. Must match a "TZ database name" shown [here](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
  2.2. Ex: Will be "America/Los_Angeles" for all of California USA.
  2.3. Ex: Will be "Europe/Dublin" for Dublin Ireland.

```javascript
alert("TimeZone: " + Intl.DateTimeFormat().resolvedOptions().timeZone);
```
